specifyweb/frontend/js_src/lib/components/FormEditor/index.tsx

```diff
@@ -53,9 +58,12 @@ export function FormEditorWrapper(): JSX.Element {
   );
 
   const [changed, setChanged] = React.useState<ReadonlySet<string>>(new Set());
+  const disciplines =
```

Comment:
I added disciplines here because we don't need the rest of content from AppResourcesOutlet. If you need additional context, maybe pass all of them down.